release/release.go

```diff
@@ -20,6 +20,57 @@ import (
 	"golang.org/x/mod/semver"
 )
 
+const (
+	k3sRepo  = "k3s"
+	rke2Repo = "rke2"
+)
+
+type changeLogData struct {
+	PrevMilestone string
+	Content       []repository.ChangeLog
+}
+
+type rke2ReleaseNoteData struct {
+	Milestone             string
+	K8sVersion            string
+	MajorMinor            string
+	EtcdVersion           string
+	ContainerdVersion     string
+	RuncVersion           string
+	MetricsServerVersion  string
+	CoreDNSVersion        string
+	ChangeLogVersion      string
+	IngressNginxVersion   string
+	HelmControllerVersion string
+	FlannelVersion        string
+	CanalCalicoVersion    string
+	CalicoVersion         string
+	CiliumVersion         string
+	MultusVersion         string
+	ChangeLogData         changeLogData
+}
+
+type k3sReleaseNoteData struct {
+	Milestone                   string
```

Comment:
If the type isn't exported, none of the fields need to be exported. This goes for the types above.

Reply:
I had to export the fields because the template needs it
```
template: release-notes:3:7: executing "k3s" at <.milestone>: milestone is an unexported field of struct type release.k3sReleaseNoteData
```

Reply:
Ah right, uses reflection under the hood like struct tags. Good catch.